### ISSUE TITLE
Teli Cleanup for @danry25

### ISSUE THREAD

**Issue by uncheckederror** [AccelerateNetworks/NumberSearch#343]
To Do for @danry25 
- [x] Update E911 Registrations for RC to send SMS notifications
- [x] Remove Teli inbound/outbound routing from FusionPBX
- [ ] Ensure Commio migrates Toll Free numbers to LCR
- [ ] Fill out form to board toll free number on 1st Point SMS/MMS
- [x] Cancel unused 10DLC campaigns migrated to Commio
- [ ] Add Commio to outbound call routing
- [x] Remove Teli-related inbound and outbound routing

_Originally posted by @danry25 in https://github.com/AccelerateNetworks/NumberSearch/issues/311#issuecomment-1528975947_

**Comment by danry25** [comment]
**Pending support work**
- [ ] Update E911 Registrations for RC to send SMS notifications
   - @sewcialite  Did this get accomplished
- [ ] Fill out form to board toll free number on 1st Point SMS/MMS
- [ ] Add Commio to outbound call routing

**Blocked by Commio**
- [ ] Ensure Commio migrates Toll Free numbers to LCR
   - Unclear if/when Commio will do this

**Comment by danry25** [comment]
Closing issue as remaining items are either blocked by vendor or not a priority
